modules/container-registry/registries/cache-rules/main.bicep

```diff
@@ -0,0 +1,18 @@
+@description('Conditional. The name of the parent registry. Required if the template is used in a standalone deployment.')
+param registryName string
+
+@description('Array of cache rules. Objects with properties: source (req), target (opt), name (opt). name and target will be derived from source if not defined.')
+param repoCaches array = []
```

Comment:
Please split into `name`, `sourceRepository` and `targetRepository` for validation purposes.

Reply:
@eriqua does that make sense given that it's an array and not an object?

Reply:
I guess this will resolve itself once the deployment of the 'array' of resources is removed.